rocky/reports/templates/partials/report_findings_table.html

```diff
@@ -0,0 +1,102 @@
+                            </td>
+                        {% else %}
+                            <td class="actions sticky-cell">
+                                <button class="expando-button"
+                                        data-icon-open-class="icon ti-chevron-down"
+                                        data-icon-close-class="icon ti-chevron-up"
+                                        data-close-label="{% translate "Close details" %}">
+                                    {% translate "Open details" %}
+                                </button>
+                            </td>
+                        {% endif %}
+                    </tr>
+                    {% if not is_dashboard_findings %}
+                        <tr class="expando-row">
+                            <td colspan="5">
+                                <h4 class="heading-normal">{% translate "Description" %}</h4>
+                                <p>{{ info.finding_type.description }}</p>
+                                <h4 class="heading-normal">{% translate "Source" %}</h4>
+                                {% if info.finding_type.source %}
+                                    <a href="{{ info.finding_type.source }}">{{ info.finding_type.source }}</a>
+                                {% else %}
+                                    <p>{{ info.finding_type.source }}</p>
+                                {% endif %}
+                                <h4 class="heading-normal">{% translate "Impact" %}</h4>
+                                <p>{{ info.finding_type.impact }}</p>
+                                <h4 class="heading-normal">{% translate "Recommendation" %}</h4>
+                                <p>{{ info.finding_type.recommendation }}</p>
+                                <h4 class="heading-normal">{% translate "Findings" %}</h4>
+                                <table class="nowrap">
+                                    <caption class="visually-hidden">{% translate "Findings overview" %}</caption>
+                                    <thead>
+                                        <tr>
+                                            <th scope="col">{% translate "Finding" %}</th>
+                                            <th scope="col">{% translate "First known occurrence" %}</th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        {% for occurrence in info.occurrences %}
+                                            <tr>
+                                                <td>
+                                                    <a href="{% ooi_url 'ooi_detail' occurrence.finding.ooi organization.code %}">{{ occurrence.finding.ooi|human_readable }}</a>
+                                                </td>
+                                                <td>{{ occurrence.first_seen|get_datetime }}</td>
```

Comment:
There's also a template function to print '2 days ago', which we can use in conjunction with a title tag to add the 'raw utc' date for power users.


See for example: https://github.com/minvws/nl-kat-coordination/blob/d72a54acb2779591c009d6f8a97fe392ea20f215/rocky/rocky/templates/oois/ooi_detail_origins_observations.html#L49C32-L49C187